models/user.js

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose';
+
+const userSchema = new mongoose.Schema({
+    fName: {
+        type: String, 
+        required: [true, 'Please type in first name'],
+        trim: true
+    }, 
+    lName: {
+        type: String, 
+        required: [true, 'Please type in last name'],
+        trim: true
+    }, 
+    email: {
+        type: String, 
+        required: [true, 'Email is required.'],
+        unique: true
+    }, 
+    password: {
+        type: String, 
+        required: [true, 'Password is required.']
+    }
+}); 
+
+module.exports = mongoose.model('User', userSchema);
```

Comment:
use es6 export 
```
export default mongoose.model('User', userSchema);
```

Reply:
why do we use es6 export in this model instead of the function used for product?

Reply:
Using ES6 export (export default) is recommended because it aligns with modern JavaScript practices, offers better performance, and ensures your code is more future-proof.